### ISSUE TITLE
have a question.

### ISSUE THREAD

**Issue by hendredlorentz** [gaasher/I-JEPA#3]
Your code is very amazing, but I wonder if this work perfectly restores the work of FAIR?

**Comment by gaasher** [comment]
I wrote this code directly from the reading the paper, so I'm not familiar with how FAIR implements their solution.  Of course, given that I'm using pytorch lightning (which is super convenient) there are probably some changes with regards to the training details (hyperparameters, LR scheduler, etc...). Do you have any section of the code in mind which you think doesn't align with FAIR's implementation?

**Comment by hendredlorentz** [comment]
Your code looks cleaner and easier to understand, I prefer to peruse your code. hah

**Comment by gaasher** [comment]
Thank you, I'm glad that you like it!
